src/Umbraco.Web/Models/ContentEditing/ContentTypeDisplay.cs

```diff
@@ -1,31 +1,28 @@
-﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.Linq;
-using System.Runtime.Serialization;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace Umbraco.Web.Models.ContentEditing
-{
-   
-    [DataContract(Name = "contentType", Namespace = "")]
-    public class ContentTypeDisplay : ContentTypeCompositionDisplay
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Umbraco.Web.Models.ContentEditing
+{
+   
+    [DataContract(Name = "contentType", Namespace = "")]
+    public class ContentTypeDisplay : ContentTypeCompositionDisplay
     {
-        public ContentTypeDisplay()
-        {
-            //initialize collections so at least their never null
-            AllowedTemplates = new List<EntityBasic>();            
-        }
-
-        //name, alias, icon, thumb, desc, inherited from the content type
-
-        // Templates
-        [DataMember(Name = "allowedTemplates")]
-        public IEnumerable<EntityBasic> AllowedTemplates { get; set; } 
-
-        [DataMember(Name = "defaultTemplate")]
-        public EntityBasic DefaultTemplate { get; set; }
-
-    }
-}
+        public ContentTypeDisplay()
+        {
+            //initialize collections so at least their never null
+            AllowedTemplates = new List<EntityBasic>();            
+        }
+
+        //name, alias, icon, thumb, desc, inherited from the content type
+
+        // Templates
+        [DataMember(Name = "allowedTemplates")]
+        public IEnumerable<EntityBasic> AllowedTemplates { get; set; } 
+
+        [DataMember(Name = "defaultTemplate")]
+        public EntityBasic DefaultTemplate { get; set; }
+
+        [DataMember(Name = "firstContentType")]
+        public bool FirstContentType { get; set; }

```

Comment:
I'm not completely sure in which case this was needed, but Mads mentioned that there is a use case where you do actually have a document type in the system, but it is not possible to use it - and in that case we needed to display a different error message than when initially creating in an empty install.

In regards to the lookup - since we are only doing the lookup when mapping from IContentType to ContentTypeDisplay (it is ignored in other cases) it will only be done when creating document types, but if you are okay with doing a separate js call for this, I like that solution better since we can decide from the UI whether to call it or not :)